src/main/java/hudson/plugins/textfinder/TextFinderPublisher.java

```diff
@@ -244,6 +243,46 @@ private static Pattern compilePattern(PrintStream logger, String regexp) {
         return pattern;
     }
 
+    /** cut off first item as it is provided with getters */
+    @SuppressWarnings("unused")
+    public List<TextFinderModel> getAdditionalTextFinders() {
+        return additionalTextFinders;
+    }
+
+    // backward compatibility getters below
+    // all these getters are there for backward compatibility
+    // get first field from list which is used to store original config values
```

Comment:
same here, no longer valid comment. we're not getting 0th element, but values from `primaryTextFinder`

Reply:
comments removed. Wil now jump to the https://github.com/jenkinsci/text-finder-plugin/pull/23#issuecomment-502392524